修改README.md 添加Nginx配置示例

YlovexLN opened the pull request.

YlovexLN committed:
docs(README): 更新模组使用说明和配置示例

- 添加了模组的功能特性说明
- 补充了前置要求和安装步骤
- 提供了Nginx反向代理配置示例
- 增加了如何连接服务器的步骤说明
- 提醒了使用注意事项

YlovexLN committed:
docs(README): 更新 Minecraft 版本要求至 1.19+

- 将 Minecraft 版本要求从 1.16+ 提升到 1.19+
- 更新 README.md 中的相关文档

YlovexLN committed:
docs(README): 更新项目许可证信息

- 将 README.md 中的 MIT 许可证标识替换为 GPL-3.0
- 更新对应的许可证文件链接

YlovexLN committed:
docs: 更新项目徽章并添加中文说明

- 将 GPL 许可证徽章从 3.0 版本更改为通用版本
- 添加了 MISS 模组的中文说明，介绍其功能和适用版本

YlovexLN committed:
docs: 更新项目许可证徽章

将 GPL 许可证徽章替换为 GPL 3.0 许可证徽章，以更准确地反映项目的许可证类型。

YlovexLN commented:
还不会分支的操作 只改了主分支的README.md

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+
+## 前置要求
+
+- Minecraft Java版 1.19+
```
```suggestion
- Minecraft Java 版 1.19+
```

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+
+- Minecraft Java版 1.19+
+- Nginx 1.18+ 
+- 有效域名与SSL证书
```
```suggestion
- 有效域名与SSL证书 (可选)
```

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+
+#### 服务端
+
+1. 从[Modrinth](https://modrinth.com/mod/miss/versions)下载对应版本
+2. 将文件放入服务端`mods`目录
+3. 重启Minecraft服务器
+
+#### 客户端
+
+1. 从[Modrinth](https://modrinth.com/mod/miss/versions)下载对应版本
+2. 将文件放入客户端`mods`目录
+3. 启动Minecraft游戏
```
```suggestion
1. 从 [Modrinth](https://modrinth.com/mod/miss/versions) 下载对应版本
2. 将文件放入服务端 `mods` 目录
3. 重启 Minecraft 服务端

#### 客户端

1. 从 [Modrinth](https://modrinth.com/mod/miss/versions) 下载对应版本
2. 将文件放入客户端 `mods` 目录
3. 启动 Minecraft 客户端
```

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+    ssl_certificate /path/to/fullchain.pem;
+    ssl_certificate_key /path/to/privkey.pem;
+
+    location / {
```
该 location 其实也可以使用不同的 path 转发到不同的服务器

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+1. 安装客户端MOD后
+2. 启动游戏
+3. 添加服务器并在地址栏输入：
+   `wss://mc.example.com:443`
```
```suggestion
   `wss://mc.example.com`
```

zyxkad commented on README.md:
```diff
@@ -1,5 +1,76 @@
+
+## 如何连接服务器
+
+1. 安装客户端MOD后
```
```suggestion
1. 安装客户端 Mod 后
```

zyxkad commented:
感谢 🙏

分支不用担心，应该没人看分支里的 README，需要的时候我也能手动 merge。

YlovexLN committed:
docs(README): 更新文档以简化部署流程

- 修改 Minecraft 和 Nginx 配置说明，使其更简洁
- 调整域名和 SSL 证书要求，标明为可选项
- 优化客户端和服务端安装说明的格式
- 移除 Nginx 配置代码中的注释
- 简化服务器连接步骤说明

YlovexLN commented:
我在我仓库改好了
